### ISSUE TITLE
Alerts for Redis unavailability

### ISSUE THREAD

**Issue by msavin** [cult-of-coders/redis-oplog#180]
It would be cool if this package created some kind of publication or something to notify clients if Redis is unavailable. Ideally, we could run something like `Meteor.hasRedis` and notify end-users of issues if there are any.

**Comment by msavin** [comment]
Additionally, maybe create room for a callback:
```javascript
RedisOplog.onRetry(function (retryCount) {
  if (retryCount > 5) {
    panic()
  }
})
```

**Comment by theodorDiaconu** [comment]
@msavin very good idea

**Comment by theodorDiaconu** [comment]
@msavin this is going to be fixed in the 1.2.3 release I already did it.
